app/scripts/dcmWalkingRetargetingVirtualizerXsens.xml

```diff
@@ -0,0 +1,181 @@
+    <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_LEFT --name /icub/camcalib/left --useIMU</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+  <module>
+    <name>camCalibWithPose</name>
+    <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_RIGHT --name /icub/camcalib/right --useIMU</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters>--device transformServer --ROS::enable_ros_publisher 0 --ROS::enable_ros_subscriber 0</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice ovrheadset --sticks 0  --tfDevice transformClient --tfLocal /oculustf --tfRemote /transformServer --tf_head_frame headoculus --tf_left_hand_frame loculus --tf_right_hand_frame roculus --tf_root_frame oculusworld --stick_as_axis true --gui_elements 0</parameters>
+    <node>icub-virtualizer</node>
+  </module>
+
+
+  <!-- Modules -->
+  <module>
+    <name>WalkingModule</name>
+    <node>icub-head</node>
+  </module>
+
+  <module>
+    <name>VirtualizerModule</name>
+    <node>icub-virtualizer</node>
+    <parameters> --context virtualizerRetargeting </parameters>
+  </module>
+
+  <module>
+    <name>OculusRetargetingModule</name>
+    <node>icub-head</node>
+    <parameters> --OCULUS::move_icub_using_joypad 0 </parameters>
+  </module>
+
+  <!-- Whole Body Retargeting using Xsens-->
+   <module>
+     <name>XsensRetargetingModule</name>
+     <node>icub-head</node>
+   </module>
+
+
+  <!-- Connections -->
+  <!-- Hand retargeting  -->
+  <connection>
+    <from>/oculusRetargeting/leftHandPose:o</from>
+    <to>/walking-coordinator/leftHandDesiredPose:i</to>
+  </connection>
+
+  <connection>
+    <from>/oculusRetargeting/rightHandPose:o</from>
+    <to>/walking-coordinator/rightHandDesiredPose:i</to>
+  </connection>
+
+  <!-- Virtualizer and Oculus -->
+  <connection>
+    <from>/walking-coordinator/torsoYaw:o</from>
+    <to>/virtualizer/robotOrientation:i</to>
+  </connection>
+
+  <connection>
+    <from>/walking-coordinator/torsoYaw:o</from>
+    <to>/oculusRetargeting/robotOrientation:i</to>
+  </connection>
+
+  <connection>
+    <from>/oculusRetargeting/walkingRpc</from>
+    <to>/walking-coordinator/rpc</to>
+  </connection>
+
+  <connection>
+    <from>/oculusRetargeting/virtualizerRpc</from>
+    <to>/virtualizer/rpc</to>
+  </connection>
+
+  <connection>
+    <from>/virtualizer/walkingRpc</from>
+    <to>/walking-coordinator/rpc</to>
+  </connection>
+
+  <connection>
+    <from>/virtualizer/playerOrientation:o</from>
+    <to>/oculusRetargeting/playerOrientation:i</to>
+  </connection>
+
+  <connection>
+    <from>/oculus/headpose/orientation:o</from>
+    <to>/oculusRetargeting/oculusOrientation:i</to>
+  </connection>
```

Comment:
@GiulioRomualdi  I delete this line as well (in both application files related to Xsens), because we are not using Oculus orientation anymore, when using Xsens data.

Reply:
Instead of commenting you can remove it :smile: